cornflow-server/cornflow/config.py

```diff
@@ -93,6 +91,11 @@ class DefaultConfig(object):
     # Password rotation time in days
     PWD_ROTATION_TIME = os.getenv("PWD_ROTATION_TIME", 120)
 
+    # Cognito configuration
+    COGNITO_REGION = os.getenv("COGNITO_REGION", None)
+    COGNITO_USER_POOL_ID = os.getenv("COGNITO_USER_POOL_ID", None)
+    COGNITO_APP_CLIENT_ID = os.getenv("COGNITO_APP_CLIENT_ID", None)
```

Comment:
At least the region and the user_pool_id are only used to create an url, right? and for azure we use the tenant_id for the url, so why delete one but keep the other one?

Reply:
deleted